feature/auth/src/main/java/com/teamoffroad/feature/auth/presentation/SetNicknameScreen.kt

```diff
@@ -73,15 +74,17 @@ internal fun SetNicknameScreen(
                     modifier = Modifier
                 )
             }
-            Spacer(modifier = Modifier.padding(vertical = 176.dp))
+            Spacer(modifier = Modifier.weight(1f))
+            //키패드 돋보기or다음 누르면 키패드내려가게?
```

Comment:
```suggestion
            // TODO: 키패드 돋보기or다음 누르면 키패드내려가게?
```
a: 이렇게 하면 녹색으로 표시가 되어서 눈에 띄어요!

Reply:
귀찮아서 안했어요....